laws/src/main/scala/cats/laws/discipline/Eq.scala

```diff
@@ -12,8 +13,10 @@ object eq {
    * and comparing the application of the two functions.
    */
   implicit def function1Eq[A, B](implicit A: Arbitrary[A], B: Eq[B]): Eq[A => B] = new Eq[A => B] {
+    val sampleCnt: Int = if (Platform.isJvm) 50 else 5
```

Comment:
@ceedubs - I've no strong feelings. I just wanted to raise the question, and the build speed is a good argument as the cats end to end build has certainly gotten longer. I just wanted to raise the question. :-)